### ISSUE TITLE
[a11y] 11.1.4.3 Contrast (minimum)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1334]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

Note 1: For texts (also in the focused state) under 18 points (or
bold text below 14 point), the contrast ratio between foreground and background color should be at least
background color should be at least 4.5:1. For large fonts, the contrast ratio should be
should be 3:1 or greater.
Note 2: Since the contrast measurements in the test were carried out using screenshots
the test results may differ from the actual color values. It is
possible that the colors are displayed differently on the screenshot than on the
screen of the device. For this reason, the test results should be regarded as guide values.
should be regarded as guidelines.

### Personal Space

- ~~[ ] Both the size information and the time information on the page do not meet the contrast do not meet the contrast requirements for texts, as the contrast ratio is currently
is currently well below 4.5:1.
The text at the bottom of the page, which shows the number of directories and
files also does not meet the contrast requirements for text.~~
![image](https://github.com/owncloud/ios-app/assets/132444655/ed704054-a10c-459d-a483-d10d11d89cc4)

**-> The colors for the brighter text and the icons are using standard system-provided colors used by iOS throughout the system. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/4be989ad-5fe7-4ae4-a84a-fa41b95b5c9d) | ![Image](https://github.com/user-attachments/assets/8351a218-4bb7-4c65-9dcc-3ff5425a5d97)

- ~~[ ] The button labels (see screenshot) do not meet the contrast requirements for text.
contrast requirements for text.~~
![image](https://github.com/owncloud/ios-app/assets/132444655/0646228a-8ca2-496e-a759-a5aba9e744df)

**-> The color for the button label is using a standard system-provided color used by iOS throughout the system. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/5f080500-7b03-4416-9744-661f50890dd6) | ![Image](https://github.com/user-attachments/assets/9a25c66e-f4d6-4450-bab0-63c9680f33c4)

### Menu

- [x] The URL of the user account does not meet the contrast requirements for texts.
![image](https://github.com/owncloud/ios-app/assets/132444655/b6aa8725-dbe1-468f-bebb-c181c3c95891)

**-> As of v12.4, the color contrast is increased: **
![Image](https://github.com/user-attachments/assets/09731b7e-6177-4330-aa07-aafc24bea427)

- [x] When focusing on active buttons in the menu, such as
"Personal", the color of the text changes in such a way that the contrast
contrast requirements for texts are no longer met.
![image](https://github.com/owncloud/ios-app/assets/132444655/f9cbe36c-7953-4a7b-8cab-87617de09cee)

** -> As of v12.4, the colors have been changed to avoid this issue: **
<img width="262" alt="Image" src="https://github.com/user-attachments/assets/d9125b3e-fd03-4d71-b7ee-fa4ecebca4df" />

### Shared with me

- [x] The size specifications as well as the time specifications and texts such as "Shared
by ..." on the page do not meet the contrast requirements for texts, as the current contrast
current contrast ratio is well below 4.5:1.

- ~~[ ] The labels of the "Accept" and "Reject" buttons do not meet the contrast requirements for texts.
do not meet the contrast requirements for texts.~~
![image](https://github.com/owncloud/ios-app/assets/132444655/643ffc9e-8beb-41f4-9ea9-16ecab610f08)

**-> The color for the button label is using a standard system-provided color used by iOS throughout the system. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/23e27927-8215-4d88-adb9-acd8a2a0ae04) |  ![Image](https://github.com/user-attachments/assets/96191d4a-6402-49b5-a78c-0c4fa37b107e)

### Shared with others

- ~~[ ] All texts that begin with "Shared by ..." do not fulfill the contrast requirements
for texts, as the current contrast ratio is well below the value of 4.5:1.
![image](https://github.com/owncloud/ios-app/assets/132444655/bf8c4b22-12a4-4b09-89d6-6e35f05435dc)~~

**-> The colors for the brighter text and the icons are using standard system-provided colors used by iOS throughout the system. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/4be989ad-5fe7-4ae4-a84a-fa41b95b5c9d) | ![Image](https://github.com/user-attachments/assets/8351a218-4bb7-4c65-9dcc-3ff5425a5d97)

### Spaces 

- [x] Depending on which background image is used for Space buttons,
it could be that the contrast requirements for the labels are not met.
are not met. It is therefore best to place the labels below or above the images.
above the images.

**-> Re-arranging the text would require wider layout changes and require more of the scarce, vertical space. This issue has therefore been resolved in #1451 by adding a dark background to the text if the iOS option `Increase Contrast` is enabled.**

Spaces Grid | Spaces Root Folder
---|----
![Image](https://github.com/user-attachments/assets/9c75e4b2-c36f-4426-8c38-41167f15a429) |  ![Image](https://github.com/user-attachments/assets/f0e92894-21de-46f5-b5be-f4aff659acd4)

### Add Account

- ~~[ ] The labels of the "Cancel" and "Continue" buttons do not meet the contrast requirements for texts.
contrast requirements for texts.~~  (** -> the Add Account wizard was replaced in the meantime: #1274 **)
